### ISSUE TITLE
[Ollama AI] Request to configure/disable default ⌘+C behavior

### ISSUE THREAD

**Issue by jaredmeakin** [raycast/extensions#15015]
### Extension

https://www.raycast.com/massimiliano_pasquini/raycast-ollama

### Description

The ⌘+C shortcut to "copy answer" is not desirable default behavior.

For example, if I ask Ollama a question, and I want to copy a portion of it's response—for example a code block—I cannot use ⌘+C. Instead, I have select the block, and then right click and select Copy.

The request is, to enable the ⌘+C behavior to be configurable, or have the ability to disable it—so that native copy shortcut behavior can be used.

### Who will benefit from this feature?

Everyone who copys a portion of Ollama's response for use elsewhere.

**Comment by MassimilianoPasquini97** [comment]
Hi @jaredmeakin,
You’re absolutely right, it can be quite annoying. As far as I know, shortcuts can’t be user-configurable for extensions. Therefore, I suggest changing ⁠Cmd + C to ⁠Cmd + Shift + C.

**Comment by MassimilianoPasquini97** [comment]
@raycastbot assign me

**Comment by MassimilianoPasquini97** [comment]
@raycastbot keep this issue open

**Comment by jaredmeakin** [comment]
@MassimilianoPasquini97 - to clarify, I'm not talking about built-in Raycast functionality. I'm talking about this extensions context menu.

![image](https://github.com/user-attachments/assets/25ef23a3-af2b-457d-899a-cf279a9a794d)

The "Copy Answer" functionality makes it really annoying when you want to copy a portion of Ollama's answer. Again, in my use-case it would be a portion of a code block that I might be interested in. Not the whole body of text.

Thanks again!

**Comment by MassimilianoPasquini97** [comment]
Yes I have understood, those shortcuts cannot be user modifies as far as I know so i need to decide a new one. In my previous message, I proposed changing Cmd + C to Cmd + Shift + C. I would like to know your @jaredmeakin opinion before making the change.

**Comment by mar627** [comment]
Hello, I think your suggestion would work @MassimilianoPasquini97, then anyone could use command c on any portion of the response they highlight.

But, I also want to propose a further tweak to the menu that could help users personalise this experience. For each command, could we alter the default action? 

Why? Most of the time, I just wish to paste the ollama output (replacing my highlighted text), so I want to make my default action "paste in active app".

**Comment by MassimilianoPasquini97** [comment]
Hi @mar627,
this behavior was fixed at least from the version where Windows support was added.

**Comment by MassimilianoPasquini97** [comment]
@raycastbot close this issue
